### ISSUE TITLE
Report configuration manager should not be available for branches

### ISSUE THREAD

**Issue by chiatt** [archesproject/arches#4254]
If a user selects the root of a branch in the Graph Designer's card tree they are able to see and configure reports.  Because branches don't have reports, users should not be able to select the root of the card tree which opens the report configuration page.

**Comment by adamlodge** [comment]
I agree that report configs should not happen on branches, but you may want to do node configuration on the root node of a branch.  Therefore, I disagree that you should not be able to select the root node.

**Comment by chiatt** [comment]
@adamlodge Good point. I'll revise the ticket.

**Comment by chiatt** [comment]
Closing as this is very low priority and probably won't be addressed before migration to Vue
